edgelet/edgelet-http-workload/src/server/cert/mod.rs

```diff
@@ -30,6 +30,24 @@ mod server;
 pub use self::identity::IdentityCertHandler;
 pub use self::server::ServerCertHandler;
 
+// 5 mins
+const AZIOT_EDGE_CA_CERT_MIN_DURATION_SECS: i64 = 5 * 60;
+
+// 180 days
+const AZIOT_EDGE_CA_CERT_MAX_DURATION_SECS: u64 = 180 * 24 * 3600;
+
+// Workload CA alias
+const IOTEDGED_CA_ALIAS: &str = "iotedged-workload-ca";
```

Comment:
Why does this exist instead of using `settings.edge_ca_cert` ?

Reply:
Oops

Reply:
Fixed